examples/trigger_astropy.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
This script should deploy with `healpy`. Have you proposed it?

Reply:
An interesting idea.  Perhaps someday...